test/dsl/dsl_options.jl

```diff
@@ -1468,3 +1469,5 @@ let
         @observables X2 ~ X1
     end
 end
+
+nothing
```

Comment:
?

Reply:
maybe move it to the end of the last `let ... end` block, just to make it clear that it belongs there (at least if that is the block which returns the last output, but I presume that would bet he case?).